src/managers/EmbraceSpanSessionManager/EmbraceSpanSessionManager.ts

```diff
@@ -76,9 +85,23 @@ export class EmbraceSpanSessionManager implements SpanSessionManagerInternal {
       );
       return;
     }
-    this._sessionSpan.setAttribute(KEY_PREFIX_EMB_PROPERTIES + key, value);
+
+    const limitedSessionProperty = this._limitManager.limitSessionProperty(
+      key,
+      value
+    );
+
+    if (limitedSessionProperty === 'dropped') {
+      return;
+    }
+
+    this._sessionSpan.setAttribute(
+      KEY_PREFIX_EMB_PROPERTIES + limitedSessionProperty.key,
+      limitedSessionProperty.value
+    );
   }
 
+  // the external api doesn't include a reason, and if a users uses it to end a session, the reason will be 'user_ended'
```

Comment:
I don't see `user_ended` anywhere in the sdk ?
```suggestion
  // the external api doesn't include a reason, and if a users uses it to end a session, the reason will be 'manual'
```

Reply:
yep looks to have been out of date, added